README.md

```diff
@@ -144,6 +144,22 @@ git push -u origin master
 
 For more details about the structure of a knowledge repository, see the technical details section below.
 
+### Configuration
+
+There are two types of configuration files, one for knowledge git repos that holds posts, and another for the web application. 
```

Comment:
I think we should be consistent in calling it the `knowledge-data` side of the things and the web application (since they both technically could be in git)